@xen-orchestra/web-core/lib/components/ui/checkbox/UiCheckbox.vue

```diff
@@ -0,0 +1,396 @@
+<!-- v3 -->
+<template>
+  <!-- @TODO: Replace it by the VtsLabel component when update -->
+  <label class="form-checkbox" v-bind="wrapperAttrs">
+    <input
+      v-model="value"
+      :class="[classNames, { indeterminate: isIndeterminate }]"
+      :disabled="isDisabled"
+      type="checkbox"
+      class="ui-checkbox"
+      v-bind="attrs"
+    />
+    <span class="fake-checkbox">
+      <VtsIcon :fixed-width="false" :icon class="icon" accent="brand" />
+    </span>
+    <span v-if="slots.default" class="typo p1-regular">
+      <slot />
+    </span>
+  </label>
+  <VtsInfo v-if="slots.info" :accent>
+    <slot name="info" />
+  </VtsInfo>
+</template>
+
+<script lang="ts" setup>
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsInfo from '@core/components/info/VtsInfo.vue'
+import { useContext } from '@core/composables/context.composable'
+import { DisabledContext } from '@core/context'
+import { toVariants } from '@core/utils/to-variants.util'
+import { faCheck, faMinus } from '@fortawesome/free-solid-svg-icons'
+import { computed, type HTMLAttributes, useAttrs } from 'vue'
+
+type CheckboxAccent = 'brand' | 'success' | 'warning' | 'danger'
```

Comment:
Maybe `brand` can be updated to `info` to match the design system.
Remember to change the CSS class names as well.
```suggestion
type CheckboxAccent = 'info' | 'success' | 'warning' | 'danger'
```